src/test/datascience/kernel-launcher/remoteKernelFinder.unit.test.ts

```diff
@@ -262,28 +259,4 @@ suite(`Remote Kernel Finder`, () => {
             'Wrong live kernel returned'
         );
     });
-    test('Invalid kernels not returned', async () => {
-        const validKernel: RemoteKernelSpecConnectionMetadata = {
-            kernelSpec: python3spec,
-            baseUrl: connInfo.baseUrl,
-            kind: 'startUsingRemoteKernelSpec',
-            id: '2',
-            serverId: computeUriHash(connInfo.url)
-        };
-        const invalidKernel: RemoteKernelSpecConnectionMetadata = {
```

Comment:
This isn't a valid test, we cannot end up with 2 kernels from two different remote jupyter uris in the same cache entry. Today we only ever support one at a time.

Also removed verification of remote jupyter uris, as we want failures so user can remove them.